### PR TITLE
export colors to a js file

### PR DESCRIPTION
**Pull request by tymmesyde**

### CODE REVIEW

File: package.json [hunk 1]
```diff
@@ -13,9 +14,11 @@
...
     },
     "scripts": {
+        "postinstall": "npm run build",
         "build": "node build.js"
     },
-    "devDependencies": {
-        "color-convert": "2.0.1"
+    "dependencies": {
+        "color-convert": "2.0.1",
+        "colord": "^2.9.3"
```
Comment (nklhtv) [inline package.json:22]: we should use specific versions here since we dont have package lock

File: package.json [hunk 2]
```diff
@@ -13,9 +14,11 @@
         "url": "https://github.com/stremio/stremio-colors.git"
     },
     "scripts": {
+        "postinstall": "npm run build",
```
Comment (nklhtv) [inline package.json:17]: its ok for development, but should be removed when the pr is ready for merging

File: build.js [hunk 3]
```diff
@@ -83,6 +91,21 @@ const readme = shades
     .join('\n')
     .replace(/^/, '|PREVIEW|NAME|HSL|HEX|\n|:---:|:---:|:---:|:---:|\n');
 
+const nativeColors = {
```
Comment (nklhtv) [inline build.js:94]: should be renamed to just `colors`

**Review by nklhtv — CHANGES_REQUESTED** [review]

File: build.js (hunk 3 above)
Comment (tymmesyde) [reply build.js]: colors is already used so i named it `colorsExport`

**Review by nklhtv — APPROVED** [review]

**Pull request merged**